scripts/espresso_block_scanner.py

```diff
@@ -0,0 +1,111 @@
+### Simple live Espresso block scanner
+### Scan all blocks looking for transaction on each VM. 
+### For each found transaction, display the VM block containing that transaction, including inclusion proofs for transactions in the Espresso block.
+
+import urllib.request
+import json;
+import time;
+import datetime;
+import sys;
+host="localhost"
+port="50000"
+if len(sys.argv) > 1:
+	host=sys.argv[1]
+if len(sys.argv) > 2:
+	port=sys.argv[2]
+url="http://"+host+":"+port+"/availability/block/"
+
+# Keep track of transaction to avoid showing duplicates
+payloads = set()
+
+# Scan block by block
+block_number = 0
+while True:
+	try:
+		page = urllib.request.urlopen(url+str(block_number))
+	except urllib.error.HTTPError as e:
+		# "Block not produced yet. waiting for 1 second
+		time.sleep(1)
```

Comment:
Print something here so the script doesn't run forever without output if something is wrong? When I first tried it I kept getting 404's.

Reply:
I was thinking in the demo in which we show the rollup block. So if Ben wanted to explain it, we needed the block previously printed to stay still.

Ideas how to do better?

Reply:
I think it's okay we can leave it as is.